### PR TITLE
Migrate away from bower

### PR DESCRIPTION
**Pull request by boyewale**
Under [CI-971](https://financialtimes.atlassian.net/browse/CI-971) and following from https://github.com/Financial-Times/n-content-body/pull/145, this is to continue the process of migrating away from Bower, but updating the packages used in this components and how they are pulled in.

This removes the bower dependency and bower.json file, and moves all the information that was in the bower.json file into the package.json - such as the description, dependencies, homepage and license. Also used the latest version of all the dependencies.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,5 +22,33 @@
   "engines": {
     "node": "12.x",
     "npm": "7.x || 8.x"
+  },
+  "dependencies": {
```
Comment (GlynnPhillips) [inline package.json:26]: **Question:** Is there a reason for these to be dependencies and not peer dependencies?

File: test/integration/compile-main.test.js [hunk 2]
```diff
@@ -25,6 +25,7 @@ describe('Compile scss file', () => {
 			includePaths: [
 				path.join(BASE_PATH, 'bower_components'),
```
Comment (GlynnPhillips) [inline test/integration/compile-main.test.js:26]: **Suggestion:** Can this be removed now?

File: package.json (hunk 1 above)
Comment (GlynnPhillips) [reply package.json:26]: I see, I guess to me (an fresh set of eyes) the distinction wasn't clear. Can you explain why we need the peer dependencies?

Comment (boyewale) [reply package.json:26]: These were the peerDependencies are required by the [packages in dependencies](https://github.com/Financial-Times/origami/blob/dece7a9391c12478fb9e114821812f8d950ddaba/components/o-quote/package.json#L24).  NPM at the time complained about these missing peerDependencies. I was trying to replicate the error to add the message here, but it seems to be fine without it now. So I've removed the peerDependencies from here.

Comment (GlynnPhillips) [reply package.json:26]: Thanks Bolu, could you also double check on whats the correct approach dependencies or peer dependencies for origami dependencies. I had a feeling we should use peer dependencies as this helps to mimic the dependencies structure of bower.

**Review by GlynnPhillips — APPROVED** [review]
From what I can tell this looks good  👍

**Pull request merged**